### PR TITLE
kn-cli Ubuntu Dockerimage

### PR DESCRIPTION
**Pull request by andersla**
## Change content and motivation

Changed kn provisioner base docker image to Ubuntu from Alpine since there is no simple way to install Microsoft Azure Cli on Alpine. Also very problematic to build and install KVM plugin on Alpine compared to Ubuntu. Image now also contains about 350MB requirements and tools - so the difference between Ubuntu and Alpine on an image with this much software is only ca 10% (Alpine ca 5MB base image - Ubuntu ca 40MB).

## GitHub cross-links 

**Fixes:** 

**Docs**:

### CODE REVIEW

**Comment by mcapuccini** [comment]
Looks good! When the CI passes I'll merge.

File: .travis.yml [hunk 1]
```diff
@@ -63,7 +63,7 @@ before_install:
   - shellcheck $(find . -type f -name "*.sh")
   - shellcheck bin/*
   # check Dockerfile
-  - docker run --rm -i lukasmartinelli/hadolint < Dockerfile
+  - docker run --rm -i lukasmartinelli/hadolint hadolint --ignore DL3008 --ignore DL3015 - < Dockerfile
```
Comment (mcapuccini) [inline .travis.yml]: * We want pinned versions in apt (https://github.com/lukasmartinelli/hadolint/wiki/DL3008)
* It's a good idea to not install reccommends (--no-install-recommends)

Please put this checks back and fix the code accordingly.

File: Dockerfile [hunk 2]
```diff
@@ -49,6 +48,7 @@ RUN curl "https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terrafor
 ENV PLUGIN_OPENSTACK=0.2.2
```
Comment (mcapuccini) [inline Dockerfile]: please group versions at the beginning of the file... I accidentally merged this as it is now

**Comment by andersla** [comment]
ENV variables for plug-ins moved to top

**Pull request merged**